project.clj

```diff
@@ -47,7 +44,7 @@
                                   [puppetlabs/trapperkeeper]
```

Comment:
Adding an exclusion for websocket-client on pcp-broker above seems like it'd avoid the need to change clj-parent.